etip/trackers/templates/tracker_list.html

```diff
@@ -6,6 +6,10 @@
             <form class="form-inline" method="get">
                 <label class="sr-only" for="inlineFormTrackerName">Tracker name:</label>
                 <input type="text" class="form-control mb-2 mr-2 col-3" value="{{ filter_name }}" name="tracker_name" id="inlineFormTrackerName" placeholder="Tracker name">
+                <div class="form-check mb-2 mr-sm-2">
+                  <input type="checkbox" class="form-check-input" id="onlyCollisionCheckbox" name="only_collisions" value="true" {{ only_collisions }}/>
```

Comment:
What does `value="true"` do?

Reply:
I thought the value was needed to be a valid html, but it looks like it's not. I will change it, thanks

Reply:
I actually don't know whether it's necessary here, just being curious :).